Data/Encryption/Encrypt.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Security.Cryptography;
+using System.Runtime.CompilerServices;
+
+namespace Data.Encryption
+{
+    public class Encrypt
+    {
+        public static string ToHash(string input, HashAlgorithm algorithm)
+        {
+            byte[] bytes = algorithm.ComputeHash(Encoding.UTF8.GetBytes(input));
```

Comment:
change byte[] for var. Is unnecesary the type if I can read the type on the name of the variable

Reply:
Done